### ISSUE TITLE
check_smtp regression with -S -D

### ISSUE THREAD

**Issue by sni** [monitoring-plugins/monitoring-plugins#1955]
v2.3.3:
```
./plugins/check_smtp -H 164.68.101.6 -S -D 7,1
OK - Certificate 'mail.nierlein.de' will expire on Thu 11 Jan 2024 09:01:15 AM GMT +0000.
```

v2.3.5:
```
./plugins/check_smtp -H 164.68.101.6 -S -D 7,1
check_smtp: Set either -s/--ssl/--tls or -S/--starttls
```

Did a quick git bisect and this regression has been introduced with ead5526efa4f713e8001baed409067b0474cb72d

**Comment by RincewindsHat** [comment]
#1952

**Comment by sni** [comment]
oh, didn't see you hit that one too :-)

**Comment by sni** [comment]
fix with #1952
